test/sql/test_others/R/test_deprecated_non_pipeline_engine

```diff
@@ -8,6 +8,7 @@ select 1;
 -- !result
 update information_schema.be_configs set value=60 where name = 'base_compaction_check_interval_seconds';
 -- result:
+[REGEX].*non-pipeline engine is no longer supported since 3.2, please set enable_pipeline_engine=true.*
 -- !result
 select `value` from information_schema.be_configs where name = 'base_compaction_check_interval_seconds' limit 1;
 -- result:
```

Comment:
Modify config during gray-scale update should be tested.
If the deployment has multiple BE(s), does it work?